examples/rbfe_single.py

```diff
@@ -77,7 +83,9 @@ def wrap_method(args: Iterable[Any], fxn: callable):
 
     cmd_args = parser.parse_args()
 
-    client = CUDAPoolClient(max_workers=cmd_args.num_gpus)
+    hosts = ["0.0.0.0:2021", "0.0.0.0:2022"]
+    client = GRPCClient(hosts=hosts)
+    # client = CUDAPoolClient(max_workers=cmd_args.num_gpus)
```

Comment:
Should we make the choice of client an argparse argument for this script, or perhaps add a separate example script for CUDAPoolClient vs GRPCClient? (Or does GRPCClient replace CUDAPoolClient?)

Reply:
It should be a parameter, which is why I was thinking of dropping this commit initially and put more work in later to swap them out cleanly